databases/week3/mealsharing.sql

```diff
@@ -0,0 +1,116 @@
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (5, 5, 5, '2020-03-04', 81335698, 'Mari', 'mdd12@hotmail.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (6, 2, 6, '2020-06-13', 51322610, 'Fariba', 'fardad@gmail.dk');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (7, 1, 2, '2020-09-20', 21207768, 'Josef', 'j120@gmail.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (8, 3, 1, '2020-08-30', 40045699, 'Ali', 'a23@yahoo.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (9, 3, 1, '2020-09-01', 91770079, 'Emil', 'emil.gas@yahoo.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (10, 4, 2, '2020-11-14', 81759878, 'Erik', 'erk@yahoo.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (11, 2, 4, '2020-12-01', 31240691, 'Thomas', 'tom_43@gmail.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (12, 5, 2, '2020-10-09', 51345665, 'Tom', 'tt65_kdf@gmail.nl');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (13, 2, 4, '2020-05-22', 50045600, 'Line', 'lin_prett@gmail.com');
+INSERT INTO Reservation (idReservation, Number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES (14, 3, 5, '2020-03-19', 61025697, 'Carsten', 'casrenrasmus@hotmail.com');
+
+-- -----------------------------------------------------
+-- Table `Mealsharing`.`Review`
+-- -----------------------------------------------------
+DROP TABLE IF EXISTS `Mealsharing`.`Review` ;
+
+CREATE TABLE IF NOT EXISTS `Mealsharing`.`Review` (
+  `idReview` INT NOT NULL,
+  `title` VARCHAR(45) NULL,
+  `description` TEXT NULL,
+  `meal_id` INT NULL,
+  `stars` INT NULL,
+  `created_date` DATE NULL,
+  PRIMARY KEY (`idReview`),
+  UNIQUE INDEX `idReview_UNIQUE` (`idReview` ASC),
+  INDEX `fk_Review_1_idx` (`meal_id` ASC),
+  CONSTRAINT `fk_Review_1`
+    FOREIGN KEY (`meal_id`)
+    REFERENCES `Mealsharing`.`Meal` (`idMeal`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB;
+
+INSERT INTO Review (idReview, title, description, meal_id, stars, created_date) VALUES (1, 'Very delicious pitza', 'the pitza was very delicious and nice', 1, 5, '2020-03-14');
```

Comment:
It might be a good idea to create a few more records task: "Sort all meals by average number of stars in the reviews"

Reply:
Thank you for the comments. Done